srcpkgs/devour/template

```diff
@@ -0,0 +1,13 @@
+# Template file for 'devour'
+pkgname=devour
+version=11.1
+revision=1
+build_style=gnu-makefile
+hostmakedepends="pkgconf"
+makedepends="libX11-devel"
+short_desc="Terminal swallower"
+maintainer="Visone Selektah <victortebar@gmail.com>"
+license="GPL-2.0-or-later"
+homepage="https://github.com/Visone-Selektah/devour"
```

Comment:
Could you explain why you used your fork instead of the upstream? Compared to upstream, this has a different version - 11.1 (why?) and changed the makefile "for void". We should try to use upstream and include a patch in srcpkgs/devour/patches if necessary.

Reply:
I use my own fork cause the origin Makefile do not work in void, so I would have to add a pre funtcion to edit it, so I forked it to use my own file.
About the patch, it do not apply in the current state of the repo. I'm working on it so i can be added soon